mappings/publisher_backlist.py

```diff
@@ -59,9 +59,18 @@ def format_authors(self):
     def format_identifiers(self):
         if 'isbn' in self.record.identifiers[0]:
             isbn_string = self.record.identifiers[0].split('|')[0]
+            print(isbn_string)
+            isbns = []
+
             if ';' in isbn_string:
                 isbns = isbn_string.split('; ')
+                
+            if ',' in isbn_string:
```

Comment:
Hmm should this be an elif? Would there be a case where isbns are both delimited by ; and , ?

Reply:
It can be an elif. I'm just used to having to use else when using elif so I chose to use two ifs instead. I can change it to elif however.